python/GafferSceneUI/SelectionToolUI.py

```diff
@@ -111,9 +111,10 @@ class SelectModePlugValueWidget( GafferUI.PlugValueWidget ) :
 
 	def __init__( self, plugs, **kw ) :
 
-		self.__menuButton = GafferUI.MenuButton( "", menu = GafferUI.Menu( Gaffer.WeakMethod( self.__menuDefinition ) ) )
+		with GafferUI.Frame( borderWidth = 1, borderStyle = GafferUI.Frame.BorderStyle.None_, **kw ) as outerFrame :
```

Comment:
It seems wrong to be doing this here, in a widget that shouldn't really care about the layout it is in. One alternative might be to give `_LeftSpacer` a height to match the highest thing. That seems more appropriate since the spacer is more about fiddling the details of the layout as a whole. Did you try reducing the size of the "Editing..." frame instead? Maybe that looks a bit rubbish?

Reply:
Maybe not quite rubbish, but the extra breathing space around the selection tip does feel nice to me.

Here's the selection tip (gray box) with border = 4 as it is now :
![image](https://github.com/GafferHQ/gaffer/assets/7873467/10181e37-7590-4c0b-9473-5704577af4c1)

and border = 1 (largest border that doesn't cause the jump) :

![image](https://github.com/GafferHQ/gaffer/assets/7873467/048e252b-76d4-4f3a-8565-399cd05b0837)

I added some height to the `LeftSpacer`, that does seem clearer to keep the details in that widget instead of `SelectModePlugValueWidget`. The 27 pixel value I got from trial and error.